internal/nodes/avi_model_evh_nodes.go

```diff
@@ -95,6 +96,13 @@ type AviVsEvhSniModel interface {
 
 	GetVHDomainNames() []string
 	SetVHDomainNames([]string)
+
+	GeneratedModel
+}
+
+type GeneratedModel interface {
+	GetGeneratedFields() *AviVsNodeGeneratedFields
```

Comment:
Is the GeneratedModel interface used somewhere else? I think it will be better to have this added directly in the AviVsEvhSniModel instead of embedding.

Reply:
Added the methods directly in AviVsEvhSniModel.